src/CardlessCredit.php

```diff
@@ -56,4 +56,26 @@ public static function createReqParams()
             'callback_url'
         ];
     }
+
+    /**
+     * Calculate payment types
+     *
+     * @param array $params user's parameters
+     * @return array
+     * @throws ApiException
+     */
+    public static function calcPaymentTypes($params = [])
```

Comment:
Can this function be renamed `calculatePaymentTypes` instead?

Reply:
Sure! I think that'll make more sense